spec/unit/gameSpec.js

```diff
@@ -0,0 +1,86 @@
+describe("Game", function () {
+  let game;
+  let frame;
+
+  beforeEach(function () {
+    frame = { 
+      addRoll: () => {},
+      addBonusScore: () => {},
+      calcFrameTotal: () => {},
+      setFrameTotal: () => {},
+      isFinalFrameComplete: () => {},
+      isFrameComplete: () => {},
+      isStrike: () => {},
+      isSpare: () => {}
+    };
+    spyOn(frameFactory, "createFrame").and.returnValue(frame);
+    game = new Game();
+  });
+
+  describe("new instancs of game class", function () {
+    it("frame array is empty for frame instance", function () {
+      expect(game.frames).toEqual([]);
+    });
+
+    it("have current frame set to frame passed in", function () {
+      expect(game.currentFrame).toEqual(frame);
+    });
+  });
+
+  describe("method for each bowl", function () {
+    it("correctly adds 1 frame to frames array", function () {
+      spyOn(game, "_isNewFrame").and.returnValues(false, true);
```

Comment:
Spying / stubbing a method of the class under test is not good. You only should stuck/mock external classes.